pyaedt/edb_core/edb_data/padstacks_data.py

```diff
@@ -1135,6 +1152,9 @@ def set_backdrill_bottom(self, drill_depth, drill_diameter):
             Name of the drill to layer.
         drill_diameter : float, str
             Diameter of backdrill size.
+        offset : float, str
+            Optional, if different to zero, assign an offset value for the backdrill. The stub will not stop at the
+            layer. This parameter is also called Mfg stub length in Aedt.
```

Comment:
@svandenb-dev Is it `Nfg` or `Mfg`?
Above you mentioned `Nfg` but it migth be `Mfg`
Just to be sure after you will have applied all the suggestion.

Reply:
@MaxJPRey thanks it's Mfg. fixed